src/mumble/GlobalShortcut_macx.mm

```diff
@@ -424,23 +424,30 @@
 	return QString(reinterpret_cast<const QChar *>(unicodeString), len).toUpper();
 }
 
-QString GlobalShortcutMac::buttonName(const QVariant &v) {
+GlobalShortcutMac::ButtonInfo GlobalShortcutMac::buttonInfo(const QVariant &v) {
 	bool ok;
 	unsigned int key = v.toUInt(&ok);
-	if (!ok)
-		return QString();
+	if (!ok) {
+		return ButtonInfo();
+	}
 
-	if (key >= MOUSE_OFFSET)
-		return translateMouseButton(key);
-	else if (key >= MOD_OFFSET)
-		return translateModifierKey(key);
-	else {
-		QString str = translateKeyName(key);
-		if (!str.isEmpty())
-			return str;
+	ButtonInfo info;
+
+	if (key >= MOUSE_OFFSET) {
+		info.device = tr("Mouse");
+		info.devicePrefix = QLatin1String("M:");
```

Comment:
```suggestion
		info.devicePrefix = QLatin1String("M");
```
I think at least for the common prefixes we should leave away the colons :thinking:

Reply:
I agree.